src/application/redux/sagas/updater.ts

```diff
@@ -50,6 +51,9 @@ import { defaultPrecision } from '../../utils/constants';
 import { updateTaxiAssets } from '../actions/taxi';
 import { periodicTaxiUpdater, periodicUpdater } from '../../../background/alarms';
 
+const MAX_ADDRESSES_TX_GENERATOR = 50;
+const MAX_ADDRESSES_UTXO_GENERATOR = 100;
```

Comment:
I would use like in the description.
const MAX_ADDRESSES_TX_GENERATOR = 15;
const MAX_ADDRESSES_UTXO_GENERATOR = 50;

On Tdex app 50 txs in one go is still too high

Reply:
I've increased the max addresses in generators due to latest change on batch server (where we batch also the /hex endpoint), @Janaka-Steph did u try to bump ldk using that change?

PS: `MAX_ADDRESSES_*` != number of txs/utxos -> in fact we don't know how many transactions and/or unspents are associated to an address so the performance also depends on wallet state.

Reply:
@Janaka-Steph

Reply:
Yes I should have use v0.5.7

Reply:
lowering that constants gives you better results? @Janaka-Steph

Reply:
Yes 15 and 50 seems ok but I still had one request failure only happening in prod. Need more testing.